man/systemd.service.xml

```diff
@@ -1088,6 +1088,279 @@ ExecStart=/bin/echo $ONE $TWO $THREE</programlisting>
   </refsect1>
 
   <refsect1>
+    <title>Service exit codes</title>
+
+    <para>The following are the basic service exit codes defined by the
```

Comment:
We should probably also say that these exit codes are returned by processes we fork off in order to start binaries, where the preparation for that fails causing us to exit with a failure. It should be mentioned that these exit codes might hence be generated from the process that usually would become the main service process.

(This is important to mention, since services of course are welcome to define their own exit codes, but we kinda live in the same "exit code namespace" with them, as it is not entirely clear whether exit code X is caused by our stub that died before it issued execve() or the actual daemon binary after the execve() succeeded...)